grpc/src/main/java/io/stargate/grpc/codec/cql/UdtCodec.java

```diff
@@ -0,0 +1,84 @@
+package io.stargate.grpc.codec.cql;
```

Comment:
Should we add copyright headers to new files? I think some other gRPC files are missing them too :thinking:

Reply:
thanks! I wish we had a plug-in for this...

Reply:
I believe we discussed using `spotless` for this, just did not get around to implementing it :)

Reply:
Fixed.

Reply:
thx! It looks like `Values.java` is still missing the header.